examples/golang-network/verify.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+export NAME=golang-network
+export UID
+export MANUAL=true
+export PORT_PROXY="${GOLANG_PORT_PROXY:-10000}"
```

Comment:
this wants to be a unique port number

i have a v hokey system for deriving them - in this case lets use `10720`